src/main/java/racing/domain/Simulator.java

```diff
@@ -7,6 +7,7 @@
 import java.util.stream.Collectors;
 
 public class Simulator {
+    private static final int MOVE_THRESHOLD = 3;
     List<Car> cars = new ArrayList<>();
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private final List<Car> cars = new ArrayList<>();
```

Reply:
적용하였습니다.